xonsh/proc.py

```diff
@@ -418,3 +434,14 @@ def stdout(self):
         else:
             self._stdout = self._tpty.buffer
         return self._stdout
+
+
+def _wcode_to_popen(code):
+    '''Converts os.wait return code into Popen format.'''
```

Comment:
Can you convert the docstring to triple double quotes rather than triple single quote, please?